src/data/projectsData.ts

```diff
@@ -38,7 +38,7 @@ const projectsData: CardRounded[] = [
       text: 'Coloq.io',
       color: 'rgba(255,255,255,.7)'
     },
-    tags: ['Startup studio', 'Conversational agents', 'IaaS'],
+    tags: ['Startup studio', 'Conversational agents', 'IaaS', 'Own product'],
```

Comment:
Sacamos estos tags a una constante?
```suggestion
    const STARTUP_STUDIO_TAG = 'Startup studio'
    const CONVERSATIONAL_AGENTS_TAG = 'Conversational agents'
    const IAAS_TAG = 'IaaS'
    const OWN_PRODUCT_TAG = 'Own product'
    tags: [STARTUP_STUDIO_TAG, CONVERSATIONAL_AGENTS_TAG, IAAS_TAG, OWN_PRODUCT_TAG],
```